pkgs/games/openxray/default.nix

```
@@ -101,7 +60,7 @@ stdenv.mkDerivation rec {
     description = "Improved version of the X-Ray Engine, the game engine used in the world-famous S.T.A.L.K.E.R. game series by GSC Game World";
     homepage = "https://github.com/OpenXRay/xray-16/";
     license = licenses.unfree // {
-      url = "https://github.com/OpenXRay/xray-16/blob/xd_dev/License.txt";
+      url = "https://github.com/OpenXRay/xray-16/blob/master/License.txt";
```

Maybe it would be better to tie it to the version, just in case it changes?

```suggestion
      url = "https://github.com/OpenXRay/xray-16/blob/${version}/License.txt";
```